### ISSUE TITLE
how to setup instructions in a small readme?

### ISSUE THREAD

**Issue by naqushab** [00dani/dot-zsh#2]

**Comment by 00dani** [comment]
This and all my other `dot-` repositories are intended to be installed and managed using [`dots`, my own dotfiles management script](https://dots.00dani.me/readme).

There probably ought to be a reference to `dots` in each individual repo's documentation to make things more discoverable! These repos aren't really in a functional state for use by others directly - there's a lot of stuff hardcoded for my use rather than being customisable with variables for instance - so I'd mostly recommend grabbing individual ideas you like from my dotfiles and using them in your own, at least for the time being.

**Comment by naqushab** [comment]
Yeah. I did that and it worked out fine. Still, the repo looked good to me,
so, I asked for the readme so that others may also benefit from it. Still,
nicely organized. :)
